packages/serializer/addon/json-api.js

```diff
@@ -471,6 +471,173 @@ const JSONAPISerializer = JSONSerializer.extend({
     return dasherize(key);
   },
 
+  // SERIALIZE
+  /**
+    Called when a record is saved in order to convert the
+    record into JSON.
+
+    By default, it creates a JSON object with a key for
+    each attribute and belongsTo relationship.
+
+    For example, consider this model:
+
+    ```app/models/comment.js
+    import Model, { attr, belongsTo } from '@ember-data/model';
+
+    export default class CommentModel extends Model {
+      @attr title;
+      @attr body;
+
+      @belongsTo('user') author;
+    }
+    ```
+
+    The default serialization would create a JSON object like:
+
+    ```javascript
+    {
+      "data": {
+        "type": "comments",
+        "attributes": {
+          "title": "Rails is unagi",
+          "body": "Rails? Omakase? O_O",
+        },
+        "relationships": {
+          "author": {
+            "data": {
+              "id": "12",
+              "type": "users"
+            }
+          }
+        }
+      }
+    }
+    ```
+
+    By default, attributes are passed through as-is, unless
+    you specified an attribute type (`attr('date')`). If
+    you specify a transform, the JavaScript value will be
+    serialized when inserted into the JSON hash.
+
+    By default, belongs-to relationships are converted into
+    IDs when inserted into the JSON hash.
```

Comment:
this isn't correct for JSON:API

Reply:
ah right, i need to do another pass on the text. i only updated the code samples.